modules/stochastic_tools/src/utils/POD.C

```diff
@@ -158,6 +165,8 @@ POD::computePOD(const VariableName & vname,
   }
   MatDestroy(&mat);
   SVDDestroy(&svd);
+
+#endif
```

Comment:
```suggestion
#else
  // These variables would otherwise be unused
  (void)vname;
  (void)left_basis_functions;
  (void)right_basis_functions;
  (void)num_nodes;
  (void)energy;
#endif
```